### ISSUE TITLE
GAT Loss Function

### ISSUE THREAD

**Issue by Wangzhen-kris** [linjieli222/VQA_ReGAT#12]
Hi,
     Thanks for your great work! I have a problem with GAT. I can't find the loss function of GAT in your works, is the GAT model supervised and how is it trained in the current work?

**Comment by linjieli222** [comment]
Thank you for your question and sorry for the late reply.

Our model is only supervised with VQA classification loss. All explicit graphs are pre-extracted from either a rule-based classifier or a pretrained calssifier, so there is no need to learn graph structure here. For implicit relation, we simply treat the graph as a fully connected graph.
